executor_manager/tests/executors/test_dispatcher.py

```
@@ -29,6 +29,15 @@ def get_executor_count(self, label_selector=None):
     def get_container_address(self, executor_name):
         return {"status": "success", "base_url": "http://localhost:10001"}
 
+    def get_container_status(self, executor_name):
```

Ruff (0.14.13)

Unused method argument: `executor_name`

(ARG002)